static/img/finding-states.svg

```diff
@@ -1,72 +0,0 @@
-<title>MUTED</title>
-<ellipse fill="none" stroke="#000000" cx="114.4617" cy="-106.8" rx="45.2247" ry="18"/>
-<text text-anchor="middle" x="114.4617" y="-102.6" font-family="Times,serif" font-size="14.00" fill="#000000">MUTED</text>
-</g>
-<!-- OPEN&#45;&gt;MUTED -->
-<g id="edge2" class="edge">
-<title>OPEN-&gt;MUTED</title>
-<path fill="none" stroke="#000000" d="M114.4617,-183.9406C114.4617,-168.8515 114.4617,-150.3672 114.4617,-135.2756"/>
-<polygon fill="#000000" stroke="#000000" points="110.9618,-184.0058 114.4617,-194.0059 117.9618,-184.0059 110.9618,-184.0058"/>
-<polygon fill="#000000" stroke="#000000" points="117.9618,-134.9833 114.4617,-124.9833 110.9618,-134.9834 117.9618,-134.9833"/>
-<text text-anchor="middle" x="137.0129" y="-163.8" font-family="Times,serif" font-size="14.00" fill="#000000">Mute &amp;</text>
-<text text-anchor="middle" x="137.0129" y="-147" font-family="Times,serif" font-size="14.00" fill="#000000">Unmute</text>
-</g>
-<!-- REMOVED -->
-<g id="node4" class="node">
-<title>REMOVED</title>
-<ellipse fill="none" stroke="#000000" cx="175.4617" cy="-18" rx="58.5482" ry="18"/>
-<text text-anchor="middle" x="175.4617" y="-13.8" font-family="Times,serif" font-size="14.00" fill="#000000">REMOVED</text>
-</g>
-<!-- OPEN&#45;&gt;REMOVED -->
-<g id="edge3" class="edge">
-<title>OPEN-&gt;REMOVED</title>
-<path fill="none" stroke="#000000" d="M147.9525,-193.2641C153.9252,-188.4659 159.4974,-182.8227 163.4617,-176.4 188.035,-136.5882 185.3103,-79.9836 180.6756,-46.324"/>
-<polygon fill="#000000" stroke="#000000" points="145.7076,-190.5688 139.6731,-199.2771 149.8211,-196.2327 145.7076,-190.5688"/>
-<polygon fill="#000000" stroke="#000000" points="184.1069,-45.6084 179.1358,-36.2522 177.1873,-46.6663 184.1069,-45.6084"/>
-<text text-anchor="middle" x="214.9852" y="-111" font-family="Times,serif" font-size="14.00" fill="#000000">Remove &amp;</text>
-<text text-anchor="middle" x="214.9852" y="-94.2" font-family="Times,serif" font-size="14.00" fill="#000000">Readd</text>
-</g>
-<!-- MUTED&#45;&gt;FIXED -->
-<g id="edge4" class="edge">
-<title>MUTED-&gt;FIXED</title>
-<path fill="none" stroke="#000000" d="M100.001,-89.6785C88.7953,-76.411 73.1085,-57.8377 60.5109,-42.9222"/>
-<polygon fill="#000000" stroke="#000000" points="63.0045,-40.4503 53.8781,-35.069 57.6567,-44.9671 63.0045,-40.4503"/>
-<text text-anchor="middle" x="93.7983" y="-58.2" font-family="Times,serif" font-size="14.00" fill="#000000">Fix</text>
-</g>
-<!-- MUTED&#45;&gt;REMOVED -->
-<g id="edge5" class="edge">
-<title>MUTED-&gt;REMOVED</title>
-<path fill="none" stroke="#000000" d="M119.1019,-88.6422C122.3895,-77.9417 127.5218,-64.4899 134.8053,-54 137.8198,-49.6585 141.4638,-45.492 145.317,-41.6321"/>
-<polygon fill="#000000" stroke="#000000" points="147.9232,-43.9886 152.8743,-34.6217 143.1626,-38.8566 147.9232,-43.9886"/>
-<text text-anchor="middle" x="158.7899" y="-58.2" font-family="Times,serif" font-size="14.00" fill="#000000">Remove</text>
-</g>
-</g>
-</svg>
```

Comment:
Note to deletion of this SVG (so it is visible here for all) and other deletions - we decided to remove the state transition diagram and information from the Managing findings document as they seem not relevant for the end user of our product. (This comment can be immediately resolved.) cc: @chmccreery